knimin/lib/data_access.py

```diff
@@ -2421,14 +2421,14 @@ def _study_exists(self, study_id):
         if not self._con.execute_fetchone(sql, [study_id])[0]:
             raise ValueError('Study ID %s does not exist.' % study_id)
 
-    def _study_is_unique(self, qiita_study_id=None, title=None, study_id=None):
+    def _study_is_unique(self, qiita_study_id=None, title=None, skip_id=None):
         """Confirms that a Qiita study ID and/or a title is not duplicate
 
         Parameters
         ----------
         qiita_study_id : int
         title : str
-        study_id : int
+        skip_id : int
```

Comment:
Can you also add why/when will we need to pass those params? For example, "useful for testing".